polars/polars-lazy/polars-plan/src/dsl/dt.rs

```diff
@@ -17,6 +17,40 @@ impl DateLikeNameSpace {
             .with_fmt("strftime")
     }
 
+    pub fn truncate(self, every: &str, offset: &str) -> Expr {
+        let every = Duration::parse(every);
+        let offset = Duration::parse(offset);
+        self.0.map(
+            move |s| match s.dtype() {
+                DataType::Datetime(_, _) => {
+                    Ok(s.datetime().unwrap().truncate(every, offset).into_series())
+                }
+                DataType::Date => Ok(s.date().unwrap().truncate(every, offset).into_series()),
+                dt => Err(PolarsError::ComputeError(
+                    format!("expected date/datetime got {:?}", dt).into(),
+                )),
+            },
+            GetOutput::same_type(),
+        )
+    }
+
+    pub fn round(self, every: &str, offset: &str) -> Expr {
```

Comment:
So this one can be removed. The function implementation should be added to the `TemporalFunction::Truncate` enum following the other examples.